third-party/grafana-operator/package.json

```diff
@@ -43,7 +43,7 @@
   },
   "crd-generate": {
     "input": [
-      "https://raw.githubusercontent.com/grafana-operator/grafana-operator/v4.8.0/deploy/manifests/v4.8.0/crds.yaml"
+      "https://raw.githubusercontent.com/grafana/grafana-operator/v5.8.0/deploy/kustomize/base/crds.yaml"
```

Comment:
You can keep v4.8.0 to avoid breaking changes.

Reply:
Does keep two version of crd in same package version a good solution?
I thought after grafana 5.0 are no longer support old crd, so replace it with new one is reasonable.
And use a major version change in this package to keep current crd user not being affect.